inspec.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.3'
 
-  spec.add_dependency 'train', '~> 1.4', '>= 1.4.35'
+  # spec.add_dependency 'train', '~> 1.4', '>= 1.4.35'
```

Comment:
We should uncomment this :smile:

Reply:
Yes once train is released we will update these 2